README.md

```diff
@@ -35,104 +33,92 @@ Note: Just a heads up, the Swift Package Manager approach isn't ready for use ju
-    
-        public var transcendConsentUrl: String
-    
-        public init(transcendConsentUrl: String)
-    
-        /// The content and behavior of the view.
-        ///
-        /// When you implement a custom view, you must implement a computed
-        /// `body` property to provide the content for your view. Return a view
-        /// that's composed of built-in views that SwiftUI provides, plus other
-        /// composite views that you've already defined:
-        ///
-        ///     struct MyView: View {
-        ///         var body: some View {
-        ///             Text("Hello, World!")
-        ///         }
-        ///     }
-        ///
-        /// For more information about composing views and a view hierarchy,
-        /// see <doc:Declaring-a-Custom-View>.
-        @MainActor public var body: some View { get }
-    
-        /// The type of view representing the body of this view.
-        ///
-        /// When you create a custom view, Swift infers this type from your
-        /// implementation of the required ``View/body-swift.property`` property.
-        public typealias Body = some View
-    }
-    ```
-    
-    ### Usage of TransendConsentWebViewUI
-    
-    ```swift
+```
+    https://github.com/transcend-io/Transcend-spm-sdk.git
+```
+
+## Usage
+Please consult the documentation [here](https://docs.transcend.io/docs/consent-management/mobile-consent/ios) for more details.
+
+### Initialization of API instance 
+
+The referrence for API instance in this repository can be found [here](https://github.com/transcend-io/consent-manager-ios-sample-sdk/blob/dev/sampleSDK/sampleSDKApp.swift#L45)
+- Note: This will not render Consent Banner on UI
+```
+    import SwiftUI
     import Transcend
     
-    struct ContentView: View {
-        var body: some View {
-          // Sample Use of TranscendWebViewUI
-          Button(action: {
-              showingPopover = true
-          })
-          {
-              Image("google")
-                  .font(.system(size: 20))
-              
-          }
-          .popover(isPresented: $showingPopover) {
-              TranscendWebViewUI(transcendConsentUrl: "https://transcend-cdn.com/cm/a3b53de6-5a46-427a-8fa4-077e4c015f93/airgap.js")
-                  .foregroundColor(Color.transcendDefault)
-                  .padding()
-          }
-        }
-                            
+    // Usage
+    // completionHandler
+    let didFinishNavigation: ((Result<Void, Error>) -> Void) = { result in
+      switch result {
+        case .success():
+          // Your logic goes here
+        case .failure(let error):
+          print("Error during web view navigation: \(error.localizedDescription)")
       }
     }
-    
-    #Preview {
-        ContentView()
+    // Create TranscendCoreConfig
+    let simpleCoreConfig: TranscendCoreConfig = TranscendCoreConfig(
+      transcendConsentUrl: "https://transcend-cdn.com/cm/{Bundle_Id}/airgap.js", mobileAppId: "com.example.ios")
```

Comment:
can we maybe pull Bundle_Id and mobileAppId both out as constants and add them to the top of the file? i think we tend to do that for various ids in our code snippets (see [here](https://docs.transcend.io/docs/consent-management/configuration/loading-asynchronously#2.-create-a-script-element))